pages/blog/[category]/[slug]/index.tsx

```diff
@@ -86,4 +99,18 @@ export async function generateStaticParams() {
 	}
 }
 
+export const getServerSideProps: GetServerSideProps<BlogPostProps> = async ({ req, res }) => {
```

Comment:
Nooooo. Don't use server-side props on blog pages. This kills the SEO advantage that we have because these pages cannot now be pre-rendered.

LOGAF: 10/10

Reply:
We don't need eventing on the API calls that these pages make. We just need a page level call every time the blog page is opened.

Reply:
Cool, instead of getting the session I'll just send anonymous id, whether user is logged in or not. Is that okay?